libs/workordersidesheet/src/lib/utils-sidesheet/useNotifications.ts

```diff
@@ -0,0 +1,23 @@
+import { useContextId, useHttpClient } from '@cc-components/shared';
+import { useQuery } from '@tanstack/react-query';
+import { WorkOrderNotification } from '../types';
+
+export const useNotifications = (workOrderId: string) => {
+  const client =  useHttpClient();
+  const contextId = useContextId();
+
+  const { data, isFetching, error } = useQuery<WorkOrderNotification[], Error>({
+    queryKey: ['workOrder','notification', workOrderId],
```

Comment:
We dont invalidate querykeys so it doesnt matter but it should be

```
queryKey: ['workOrder', workOrderId, 'notification'],
```

Reply:
Fixed :)